### ISSUE TITLE
Works on Mac

### ISSUE THREAD

**Issue by PixelT** [titoBouzout/Toolbar#10]
It's possibility to work this on Mac?

**Comment by MattDMo** [comment]
Nope. See #2.

**Comment by titoBouzout** [comment]
Yes, it works, it does display some "weird" emojies, but if you ignore that, it just works.

**Comment by PixelT** [comment]
The icons are displayed, but not react (action) when I click on it...

**Comment by titoBouzout** [comment]
OK, then it does not work anymore.

**Comment by evandrocoan** [comment]
Up it for native `Sublime Text` support: https://github.com/SublimeTextIssues/Core/issues/1373
